src/modules/general/components/topBanner/index.tsx

```diff
@@ -6,37 +6,55 @@ import { FeaturedIconOutlined } from '../featuredIconOutlined';
 import { Link } from '../link';
 
 interface TopBannerProps {
-  theme: 'success' | 'warning';
+  theme: 'success' | 'warning' | 'purple';
   primaryBtnLabel?: string;
   primaryBtnAction?: () => void;
   secondaryBtnLabel?: string;
   secondaryBtnLink?: string;
   primaryBtnIcon?: ReactNode;
+  primaryButtonStyle?: string;
   secondaryBtnAction?: () => void;
   text: string;
   supportingText: string;
 }
+const colors = [
+  {
+    theme: 'warning',
+    bgColor: variables.color_warning_25,
+    borderColor: variables.color_warning_300,
+    textColor: variables.color_warning_700,
+  },
+  {
+    theme: 'success',
+    bgColor: variables.color_success_25,
+    borderColor: variables.color_success_300,
+    textColor: variables.color_success_700,
+  },
+  {
+    theme: 'purple',
+    bgColor: variables.color_purple_50,
+    borderColor: variables.color_purple_200,
+    textColor: variables.color_purple_700,
+  },
+];
+
 export const TopBanner: React.FC<TopBannerProps> = ({
   theme,
   primaryBtnAction,
   primaryBtnLabel,
   primaryBtnIcon,
+  primaryButtonStyle,
   secondaryBtnLabel,
   secondaryBtnAction,
   secondaryBtnLink,
   text,
   supportingText,
 }) => {
-  const bgColor =
-    theme === 'warning' ? variables.color_warning_25 : theme === 'success' ? variables.color_success_25 : '';
-  const borderBottomColor =
-    theme === 'warning' ? variables.color_warning_300 : theme === 'success' ? variables.color_success_300 : '';
-  const textColor =
-    theme === 'warning' ? variables.color_warning_700 : theme === 'success' ? variables.color_success_700 : '';
+  const color = colors.find(c => c.theme === theme);
   return (
     <div
-      className="w-full pb-2 p-4  xl:px-6 xl:py-1 flex flex-col xl:flex-row xl:items-center justify-between gap-2"
-      style={{ backgroundColor: bgColor, borderBottom: `1px solid ${borderBottomColor}` }}
+      className={`w-full pb-2 p-4 xl:px-6 ${theme === 'purple' ? 'xl:py-3' : 'xl:py-1'} flex flex-col xl:flex-row xl:items-center justify-between gap-2`}
```

Comment:
I knew this condition was for just a 'purple' theme but can't we tell Minh to make a consistent style for all banners?

Reply:
otherwise please define a prop for making different styles from outside of component.